### DISCUSSION TITLE
I think there is a typo in README.md on branch master.

### DISCUSSION THREAD

**Discussion by Akira-Al** [NeogitOrg/neogit#827] [General]
I think you have forgotten to put a comma at the and of line 104, in README.md on branch master.
The current code is 
`disable_line_numbers = true`
but I think it should be 
`disable_line_numbers = true,`.
I'm new to Lua so May be wrong.

**Comment by CKolkey** [comment]
I think you're right! Thanks :)

**Comment by star-szr** [comment]
I think this discussion can be closed now, this was fixed in https://github.com/NeogitOrg/neogit/commit/f124c6029397c9a12ac1a85ea63179b8a6018a31. Thanks!
